docs/source/executing-operations/subscription-support.mdx

```diff
@@ -3,9 +3,7 @@ title: GraphQL subscriptions with a self-hosted router
 description: Real-time data with GraphOS Enterprise
 ---
 
-> ⚠️ **For self-hosted routers, subscription support is an [Enterprise feature](../enterprise-features/).**
->
-> Subscription support is _also_ available for cloud routers with a GraphOS Serverless plan. [See the docs.](/graphos/operations/subscriptions)
+<GraphOSEnterpriseRequired />
```

Comment:
I think we may have to leave the current version, since the apparently users can use this feature on Serverless as well as Enterprise?